arches/app/templates/registration/password_reset_form.html

```diff
@@ -24,6 +24,6 @@ <h3 class="login-panel-header arches-signin">{% trans "Password Recovery" %}</h3
 
 	</form>
 	<div class="panel-footer">
-        Learn more about <a href="archesproject.org">{% trans "Arches" %}</a>
+        {% trans "Learn more about" %} <a href="archesproject.org">{% trans "Arches" %}</a>
```

Comment:
this whole line including tags should be wrapped in a "trans" tag, this way the translator can put the emphasis in the correct context
```{% trans "Learn more about <a href='archesproject.org'>Arches</a>" %}```